### PR TITLE
🔧(project) add missing gitlint configuration file

### PR DESCRIPTION
**Pull request by jmaupetit**
## Purpose

Our standard gitlint configuration is missing.

## Proposal

- [x] add missing `.gitlint` configuration file

### CODE REVIEW

File: .gitlint [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ignore=B6,UC1
+
+# [ignore-by-body]
+# Ignore certain rules for commits of which the body has a line that matches a regex
+# E.g. Match bodies that have a line that contains "release"
+# regex=(.*)release(.*)
+#
+# Ignore certain rules, you can reference them by their id or by their full name
+# Use 'all' to ignore all rules
+# ignore=T1,body-min-length
```
Comment (wilbrdt) [inline .gitlint:77]: You're missing the EOF carriage return
```suggestion
# ignore=T1,body-min-length

```

Comment (jmaupetit) [reply .gitlint:77]: :thinking: I don't think so. If it was the case, github would have displayed it as so. :wink:

Comment (wilbrdt) [reply .gitlint:77]: You're right, my bad!

**Review by wilbrdt — APPROVED** [review]
🎉

**Review by quitterie-lcs — APPROVED** [review]

**Pull request merged**